<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpForm</class>
 <widget class="QWidget" name="SignUpForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign Up</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelUser">
       <property name="text">
        <string>user name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditUser">
       <property name="styleSheet">
        <string notr="true">border-color: rgb(255, 89, 89);</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelPassword">
       <property name="text">
        <string>password:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditPassword">
       <property name="styleSheet">
        <string notr="true">
border-color: rgb(243, 255, 108);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelFullName">
       <property name="text">
        <string>full name: </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditFullName">
       <property name="styleSheet">
        <string notr="true">border-color:rgb(139, 255, 172)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelEmail">
       <property name="text">
        <string>email:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditEmail">
       <property name="styleSheet">
        <string notr="true">border-color:rgb(170, 85, 127)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelAge">
       <property name="text">
        <string>age:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEditAge">
       <property name="styleSheet">
        <string notr="true">border-color: rgb(65, 255, 242);</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelGender">
       <property name="text">
        <string>gender: </string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="comboBoxGender">
       <property name="currentText">
        <string>male</string>
       </property>
       <item>
        <property name="text">
         <string>male</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>female</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnSignUp">
     <property name="text">
      <string>Sign up</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
